modules/photo/src/layerModelBlending.cpp

```diff
@@ -8,37 +8,32 @@
 using namespace std;
 using namespace cv;
 
-#define EPSILON      1e-6f
-
-#define SAFE_DIV_MIN EPSILON
-#define SAFE_DIV_MAX (1.0f / SAFE_DIV_MIN)
-
-#define	CLAMP(f,min,max)	((f)<(min)?(min):(f)>(max)?(max):(f))
-
-namespace cv
-{
-/*  local function prototypes  */
-static inline float	safe_div(float a, float b);
-/* returns a / b, clamped to [-SAFE_DIV_MAX, SAFE_DIV_MAX].
- * if -SAFE_DIV_MIN <= a <= SAFE_DIV_MIN, returns 0.
- */
-static inline float safe_div(float a, float b)
-{
-    float result = 0.0f;
-    if (fabsf(a) > SAFE_DIV_MIN)
-    {
-        result = a / b;
-        result = CLAMP(result, -SAFE_DIV_MAX, SAFE_DIV_MAX);
-    }
-    return result;
-}
+#define LayerModelBlend_Darken(A,B)     ((uchar)((B > A) ? A:B))
```

Comment:
If you want to copy code from StackOverflow, you should give a credit (e.g. comment with a link).